docs/en/compute/AWS-EMR.md

```diff
@@ -28,59 +28,74 @@ different cloud provider's storage i.e. GCS, Azure Blob Store.
-the root UFS can be reconfigured to be HDFS.
+[AWS EMR Getting Started](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-gs.html) guide.
+An S3 bucket is needed as Alluxio's Root Under File System and to serve as the location for the
+bootstrap script.
+If required, the root UFS can be reconfigured to be HDFS.
 
 ## Basic Setup
 
 To begin with, [download an Alluxio release](https://www.alluxio.io/download) and unzip it.
 
 1. Set up the required IAM roles for the account to be able to use the EMR service.
-```bash
-aws emr create-default-roles
+```console
+$ aws emr create-default-roles
 ```
 2. Download `alluxio-emr.sh` from [our github](https://github.com/Alluxio/alluxio/tree/master/integration/emr/) and
 upload the script to your S3 bucket.
-3. The input arguments for the bootstrap script are in the order below:
-    - The URI from where to download the Alluxio Release `.tar.gz` file. This can be an `http://`
-     URI, `s3://` URI or an `http://` URI.
-     This is a mandatory property.
-    - The root-ufs-uri. This should be an `s3://` or `hdfs://` URI designating the root mount of the Alluxio file system.
-       This is a mandatory property.
-    - Extra alluxio options. These are specified as a comma-separated list of key-values in the format `<key>=<value>`.
-       For example, `alluxio.user.file.writetype.default=CACHE_THROUGH`
-```
-aws emr create-cluster \
+3. There are 5 input arguments for the bootstrap script. They are described below:
+    - (Mandatory) The URI from where to download the Alluxio Release `.tar.gz` file.
+    This can be an `https://` URI, `s3://` URI or an `http://` URI.
+    - (Mandatory) The root-ufs-uri.
+      This should be an `s3://` or `hdfs://` URI designating the root mount of the Alluxio file
+      system.
+    - (Optional) A path path to a remote `alluxio-site.properties` that is used during the bootstrap process.
+      Properties in this file will be set on all master+worker nodes.
+      Can be set to empty string (`""`) to ignore.
+    - (Optional) Extra alluxio options.
+      These are specified as a delimited list of key-values in the
+      format `<key>=<value>`.
+      The value for the delimiter is `;` by default.
+      It can be changed by setting the 5th argument.
+      For example, `alluxio.user.file.writetype.default=CACHE_THROUGH`
+    - (Optional) A delimiter value for additional properties.
+      Empty string (`""`) defaults to `;`.
+      This property can be left out.
+```console
+$ aws emr create-cluster \
 --release-label emr-5.23.0 \
 --instance-count <num-instances> \
 --instance-type <instance-type> \
 --applications Name=Presto Name=Hive Name=Spark \
 --name '<cluster-name>' \
 --bootstrap-actions \
 Path=s3://bucket/path/to/alluxio-emr.sh,\
-Args=[<download-url>,<root-ufs-uri>,<additional-properties>] \
+Args=[<download-url>,<root-ufs-uri>,<path>,<path-to-site-properties>,<additional-delimited-properties>,<property-delimiter>] \
```

Comment:
- remove `<path>`? You are taking 6 args where 5 are expected
- How about `<optional-path-to-site-properties>`, `<optional-additional-properties>`,`<optional-property-delimiter>`